containers/reverse-proxy/envoy-config-template.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+cat << EOF
```

Comment:
Let's inline this in the envoy-start script. You can do the output redirect like this:

```
cat > /etc/envoy/envoy.yaml << EOF
...
```

Reply:
Resolved